bindings/pyroot/pythonizations/test/rdf_filter.py

```diff
@@ -0,0 +1,123 @@
+import unittest
+import ROOT
+import numpy as np
+
+# These imports are specific to test_with_dtypes
+import random 
+import pandas as pd
+import os
+from helper import *
+
+class PyFilter(unittest.TestCase):
+    """
+    Testing Pythonized Filters of RDF
+    """
+
+    def test_with_dtypes(self):
+        """
+        Tests the pythonized filter with all the tree datatypes and 
+        """
+        CreateData()
+        rdf = ROOT.RDataFrame("TestData", "./TestData.root")
+        test_cols =[str(c) for c in rdf.GetColumnNames()]
+        df = pd.DataFrame(rdf.AsNumpy())
+        for col_name in test_cols:
+            filter = "filter_" + TYPE_TO_SYMBOL[col_name] # string of filter name
+            func = eval(filter); # function of the filter
```

Comment:
Maybe rather than using this `eval` pattern, you could have a dictionary mapping the `col_name` to its respective function:

```
{"B": filter_B, "C": filter_C}
```